Firestore/core/src/firebase/firestore/remote/remote_objc_bridge.h

```diff
@@ -71,8 +71,8 @@ class WatchStreamSerializer {
   nanopb::Message<google_firestore_v1_ListenRequest> EncodeUnwatchRequest(
       model::TargetId target_id) const;
 
-  nanopb::MaybeMessage<google_firestore_v1_ListenResponse> ParseResponse(
-      const grpc::ByteBuffer& buffer) const;
+  nanopb::Message<google_firestore_v1_ListenResponse> ParseResponse(
```

Comment:
Ideally the read method that's called on this would be virtual such that any `Reader*` could be supplied here, no?